examples/src/main/java/com/mapbox/navigation/examples/core/SimpleMapboxNavigationKt.kt

```diff
@@ -346,7 +345,7 @@ class SimpleMapboxNavigationKt : AppCompatActivity(), OnMapReadyCallback,
             isAlternativeFaster: Boolean
         ) {
             if (isAlternativeFaster) {
-                this@SimpleMapboxNavigationKt.fasterRoute = fasterRoute
+                this@SimpleMapboxNavigationKt.fasterRoute = currentRoute
```

Comment:
may I know why we need this change if it is intentionally?

Reply:
Someone forgot to change it when refactor this code. Because fasterRoute=fasterRoute doesn't have any sense) And I checked commit history - before it was exactly `currentRoute`)

Reply:
looks like it's here:
https://github.com/mapbox/mapbox-navigation-android/pull/2780/files#diff-4f1754f22f1ab747ace2ce6e3d194163L136-L139

but it's been fixed anyway. Thanks.